include/AudioEngine.h

```diff
@@ -235,9 +235,18 @@ class LMMS_EXPORT AudioEngine : public QObject
 	}
 
 
-	sample_rate_t baseSampleRate() const;
-	sample_rate_t outputSampleRate() const;
-	sample_rate_t inputSampleRate() const;
+	sample_rate_t baseSampleRate() const { return m_baseSampleRate; }
+
+
+	sample_rate_t outputSampleRate() const{
+		return m_audioDev != nullptr ? m_audioDev->sampleRate() : m_baseSampleRate;
+	}
+	
+
+	sample_rate_t inputSampleRate() const{
+		return m_audioDev != nullptr ? m_audioDev->sampleRate() : m_baseSampleRate;
+	}
+
```

Comment:
```suggestion
	sample_rate_t outputSampleRate() const { return m_audioDev != nullptr ? m_audioDev->sampleRate() : m_baseSampleRate; }
	sample_rate_t inputSampleRate() const { return m_audioDev != nullptr ? m_audioDev->sampleRate() : m_baseSampleRate; }
```
Just some formatting changes.

Reply:
Do we really need that? It looks very long.

Reply:
Fair, maybe just put the braces on there own line? If not it's okay I wont nitpick, its just that the formatting looked strange to me a bit.